### PR TITLE
NOTIF-487 Using pagination for event types

### PR DESCRIPTION
**Pull request by josejulio**
Wraps event_types response in a page object, to be able to know the total number of elements.

### CODE REVIEW

**Comment by josejulio** [comment]
@gwenneg could you review? It's still a draft because it needs to go together with https://github.com/RedHatInsights/notifications-frontend/pull/240

File: backend/src/main/java/com/redhat/cloud/notifications/db/Query.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 public class Query {
 
     private static final Pattern SORT_FIELD_PATTERN = Pattern.compile("^[a-z0-9._-]+$", CASE_INSENSITIVE);
+    private static final int DEFAULT_RESULTS_PER_PAGE  = 20;
```
Comment (gwenneg) [inline backend/src/main/java/com/redhat/cloud/notifications/db/Query.java:13]: This should probably be mentioned in the OpenAPI doc for all affected endpoints.

Comment (josejulio) [reply backend/src/main/java/com/redhat/cloud/notifications/db/Query.java:14]: Added the `@DefaultParam` annotation, that updates the openapi.json

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**